src/backend/src/routes/slack.routes.ts

```diff
@@ -0,0 +1,12 @@
+import { createEventAdapter } from '@slack/events-api';
+import SlackController from '../controllers/slack.controllers';
+
+export const slackEvents = createEventAdapter(process.env.SLACK_SIGNING_SECRET || '');
+
+slackEvents.on('message', async (event) => {
+  SlackController.processMessageEvent(event);
+});
+
+slackEvents.on('error', (error) => {
+  console.log(error.name);
```

Comment:
can we also log the stringified error for more info

Reply:
ok, I will also do that for our controller try-catch to catch errors in our code